tests/error/error_types.cpp

```diff
@@ -1,61 +0,0 @@
-/*******************************************************************************
-//
-//  SYCL 2020 Conformance Test Suite
-//
-//  Copyright:	(c) 2017 by Codeplay Software LTD. All Rights Reserved.
-//
-*******************************************************************************/
-
-#include "../common/common.h"
-
-#define TEST_NAME error_types
-
-namespace error_types {
-using namespace sycl_cts;
-
-/**
- */
-class TEST_NAME : public util::test_base {
- public:
-  /** return information about this test
-   */
-  void get_info(test_base::info &out) const override {
-    set_test_info(out, TOSTRING(TEST_NAME), TEST_FILE);
-  }
-
-  /** execute this test
-   */
-  void run(util::logger &log) override {
-#define CHECK_EXISTS(EXCEPTION_NAME)                       \
-  if (!std::is_class<EXCEPTION_NAME>::value) {             \
-    FAIL(log, "EXCEPTION_NAME is not defined as a class"); \
-  }
-
-    CHECK_EXISTS(sycl::exception);
```

Comment:
I would expect these checks to be replaced with checks for different error codes existing, is that already somewhere or is it planned for a later PR?

Reply:
Yes, tests for `enum class errc` with error codes are going to be added in later PR on this week.